---
float32[] gripper_state
